datadog_lambda/tracing.py

```diff
@@ -202,7 +224,9 @@ def create_sns_event(message):
     }
 
 
-def extract_context_from_sqs_or_sns_event_or_context(event, lambda_context):
+def extract_context_from_sqs_or_sns_event_or_context_and_set_dsm_ckpt_if_enabled(
```

Comment:
I'm fine to just leave this method name what it was. I think the comment here suffices.